getting-started/add-test-and-security.hbs.md

```diff
@@ -269,6 +269,13 @@ Verify that both Scan Controller and Grype Scanner are installed by running:
 
     Where `VERSION-NUMBER` is your Tanzu Application Platform version. For example, `{{ vars.tap_version }}`.
 
+>**Note** In Tanzu Application Platform v1.7 there is [a known issue](../release-notes.hbs.md#1-7-0-scst-scan-ki) leading to a situation where the default Trivy Scanner Image is pointing to an unaccessible location. One can correct this by setting:
```

Comment:
I don't think we should put this here since Scan 2.0 is not the default scanner and would not be in the "Getting Started" path.  This should likely be moved to [here](https://github.com/pivotal/docs-tap/blob/main/scst-scan/integrate-app-scanning.hbs.md?plain=1#L35)